content/en.json

```
@@ -92,13 +88,13 @@
       "subtitle": "We work in a modern way, aiming for quality. \"First time right\" is also cheapest in the end.",
       "how-title": "How we like to do this:",
       "how": [
-        "Modern version control",
-        "Up to date tooling, frameworks, and libraries",
+        "Everything in version control",
+        "Up-to-date tools",
```

Personally I wouldn't think "frameworks and libraries" are included if you only mention "tools", so in that case we'd loose information? Id' suggest:

```suggestion
        "Up-to-date tools, frameworks, and libraries",
```

or if we really want some extra brevity:


```suggestion
        "Up-to-date tools and libraries",
```

---

I did intend for "tools" to be an umbrella term for "stuff we use". But if that doesn't come across, sure, let's keep frameworks and libraries.